app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="row vh-100 d-flex">
+    <div class="col-md-6 mx-auto">
+      <div class="card">
+        <div class="card-body">
+          <h2 class="card-title text-center">Sign up</h2>
+
+          <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+            <%= render "devise/shared/error_messages", resource: resource %>
+
+            <div class="form-group mb-3">
+              <%= f.label :email %>
+              <%= f.email_field :email, class: "form-control", autofocus: true, autocomplete: "email" %>
+            </div>
+
+            <div class="form-group mb-3">
+              <%= f.label :password %>
+              <% if @minimum_password_length %>
+                <em>(<%= @minimum_password_length %> characters minimum)</em>
+              <% end %>
+              <%= f.password_field :password, class: "form-control", autocomplete: "new-password" %>
+            </div>
+
+            <div class="form-group mb-3">
+              <%= f.label :password_confirmation %>
+              <%= f.password_field :password_confirmation, class: "form-control", autocomplete: "new-password" %>
+            </div>
+
+            <div class="form-group mb-3">
+              <%= f.submit "Sign up", class: "btn btn-primary" %>
+            </div>
```

Comment:
- [ Required ] As requested by the previous reviewer, I think it would be nice if you made the `Sign Up` button have the same width as the other input elements which take the `full width` of the login form. Currently, the button has a smaller width compared to the other input elements. Please see the screenshot and code suggestion below and rectify this.

## Example code
```ruby
   <div class="form-group mb-3">
      <%= f.submit "Sign up", class: "btn btn-primary w-100" %>
  </div>
```

## Sign Up form
![Screenshot from 2023-11-19 00-00-30](https://github.com/MenendezON/Blog-app/assets/16392046/cdc7ba00-5d6d-42ea-bf0d-c533d14d8a12)